packages/maskbook/src/plugins/Collectible/UI/ListingByHighestBidCard.tsx

```diff
@@ -142,12 +143,11 @@ export function ListingByHighestBidCard(props: ListingByHighestBidCardProps) {
                 <DateTimePanel
                     label={t('plugin_collectible_expiration_date')}
                     date={expirationDateTime}
+                    min={formatDateTime(new Date(), "yyyy-MM-dd'T00:00")}
```

Comment:
Is this correct that `expiration_date` is less than current time?

Reply:
Right, it may be early than current time. I would change it to `"yyyy-MM-dd'T00:00"`

Reply:
I changed it to `"yyyy-MM-dd'Thh:mm"`